vmdb/app/models/container_node.rb

```diff
@@ -8,4 +8,10 @@ class ContainerNode < ActiveRecord::Base
   belongs_to :lives_on, :polymorphic => true
 
   delegate   :hardware, :to => :computer_system
+
+  virtual_column :condition_ready, :type => :string, :uses => :container_node_conditions
+
+  def condition_ready
+    container_node_conditions.find_by_name('Ready').status
```

Comment:
@h-kataria I somehow missed the *not* in my last comment (now fixed), so to my understanding - there is not a possibility that container_node_conditions.find_by_name('Ready') is nil.

Do you prefer I make sure it's not nil and if indeed nil return "None"?